api/src/test/java/org/pmiops/workbench/workspaceadmin/WorkspaceAdminControllerTest.java

```diff
@@ -66,12 +69,20 @@
   private static final String WORKSPACE_NAME = "name";
 
   @TestConfiguration
-  @Import({WorkspaceAdminController.class, WorkspaceMapperImpl.class})
+  @Import({WorkspaceAdminController.class, WorkspaceMapperImpl.class, CommonMappers.class})
   @MockBean({
     CloudStorageService.class,
     NotebooksService.class,
   })
-  static class Configuration {}
+  static class Configuration {
+    @Bean
+    WorkbenchConfig workbenchConfig() {
```

Comment:
I thought @gjuggler had a scheme where you could extend a parent class and get a config for free. I'm too hungry to go look it up or how to use it. This looks OK to me, but there's some duplication across test classes.